### ISSUE TITLE
Add component status to documentation

### ISSUE THREAD

**Issue by HeleenSG** [minvws/nl-rdo-manon#801]
### Describe the bug, issue or concern

Users should be able to see the current state of a component. Has it been checked on: ux, privacy, security and accessibility. Are there any known issues? Is the component still work in progress or is it a stable release. 

This should be added to the documentation.

### Todo
- [ ] Add state to all components
- [ ] Add known issues

**Comment by HeleenSG** [comment]
Moving this back to the backlog as a design is needed.

**Comment by HeleenSG** [comment]
Finished with manon v18 update
